lib/utils/is-electron.ts

```diff
@@ -0,0 +1,9 @@
+// https://github.com/atom/electron/issues/22
+
+export const isElectron: () => boolean = () => {
+  debugger;
+  return !!window?.process?.type;
+};
+
+export const isMacElectron: () => boolean = () =>
+  window?.process?.platform === 'darwin';
```

Comment:
what if we called this `lib/utils/platform.ts` since it carries more information than just Electron?